docs/UserGuide.md

```diff
@@ -10,60 +10,70 @@ While we were applying to internships in our second year, we quickly realised ho
 various Excel sheets and Word documents, so SOCareers was born!
 
 SOCareers is your ultimate companion to managing your professional connections and internship applications.
-Designed with School of Computing (SoC) students in mind, it is optimised for use via a
-Command Line Interface (CLI). This enables you to organise your information more efficiently than traditional Graphical
+Designed with School of Computing (SoC) students in mind, it is optimised for use via a Command Line Interface (CLI).
+If you can type fast, SOCareers enables you to organise your information more efficiently than traditional Graphical
 User Interface (GUI) apps.
 
 ## How to use our guide
+Our user guide provides the format of every command available in SOCareers as well as
+examples to show you how to use each command.
+
 If you are a new user, we recommend you look through our [Quick Start](#quick-start) guide. With installation
 instructions, an overview of our GUI and a tutorial on how to use the CLI, it has everything to get you acquainted with
 SOCareers!
 
 If you are an experienced user, you can use the [Table of Contents](#table-of-contents) to easily find guides on your
```

Comment:
Same here do you wanna put this in point form?

Reply:
Sure